File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Convert Word to PDF in Windows Forms | Syncfusion 
+description: Convert Word document to PDF in Windows Forms application using Syncfusion .NET Word (DocIO) library without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word document to PDF in Windows Forms
+
+Syncfusion DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to **create, read, edit and convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert Word document to PDF in Windows Forms**.
+
+## Steps to convert Word document to PDF programmatically:
+
+Step 1: Create a new Windows Forms application project.
+
+Step 2: Install [Syncfusion.DocIO.WinForms](https://www.nuget.org/packages/Syncfusion.DocIO.WinForms/) NuGet package as a reference to your Windows Forms application from the [NuGet.org](https://www.nuget.org/).
```

Comment:
For Word to PDF, Syncfusion.DocToPdfConverter.WinForms NuGets to use.

Reply:
Addressed